### PR TITLE
Adds a cool new admin-only mob!

### PR DESCRIPTION
**Pull request by SgtHunk**
Adds a cool new pony mob that's awesome and here it is
This improves Fulpstation to its fullest
(Disclaimer: Sprites not mine, credits to the 2013 Ponystation, rest in peace.)
imagen
imagen

Here's all the names in case you want to change the icon states
imagen

### CODE REVIEW

**Comment by StoneMason** [comment]
please no

File: fulp_modules/features/mobs/pony.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"pinkie",
+	"rainbow",
+	"rarity",
+	"tia",
+	"trixie_a_cape",
+	"trixie_a_full",
+	"twilight",
+	"vinyl",
+	"whooves",
+	"pony",
```
Comment (JohnFulpWillard) [inline fulp_modules/features/mobs/pony.dm:42]:
```suggestion
		"applejack",
		"clownie",
		"fleur",
		"fluttershy",
		"luna",
		"lyra",
		"mac",
		"pinkie",
		"rainbow",
		"rarity",
		"tia",
		"trixie_a_cape",
		"trixie_a_full",
		"twilight",
		"vinyl",
		"whooves",
		"pony",
```

File: fulp_modules/features/mobs/pony.dm [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/mob/living/simple_animal/hostile/pony
+	name = "Pony"
+	desc = "Is that a fucking Pony?"
+	icon = 'fulp_modules/features/mobs/pony.dmi'
+	icon_state = "pony"
+	gender = NEUTER
+	health = 150
+	maxHealth = 150
+	obj_damage = 60
+	melee_damage_lower = 20
```
Comment (JohnFulpWillard) [inline fulp_modules/features/mobs/pony.dm:10]: wtf

Comment (SgtHunk) [reply fulp_modules/features/mobs/pony.dm:10]: admin request they hit like a truck

File: fulp_modules/features/mobs/pony.dm (hunk 1 above)
Comment (SgtHunk) [reply fulp_modules/features/mobs/pony.dm:42]: 😕

**Comment by HugoOdaX** [comment]
Speedmerge

**Comment by Horatio22** [comment]
I think I speak for the whole admin team when I say how much we truly appreciate this, Pucci

**Comment by Pepsilawn** [comment]
fulpstation just got 20% cooler /)

**Comment by Sheits** [comment]
please...

**Comment by Ebb-Real** [comment]
cursed cursed cursed cursed cursed cursed cursed cursed

**Comment by wraith-54321** [comment]
oh no

**Comment by SoreYew** [comment]
I know what to pray for now

File: fulp_modules/features/mobs/pony.dm [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/mob/living/simple_animal/hostile/pony
+	name = "Pony"
+	desc = "Is that a fucking Pony?"
```
Comment (ATH1909) [inline fulp_modules/features/mobs/pony.dm:3]:
```suggestion
	desc = "Is that a fucking pony?"
```

Comment (ATH1909) [reply fulp_modules/features/mobs/pony.dm:3]: unless mlp has some weird capitalization related to the word "pony" that I don't know about

File: fulp_modules/features/mobs/pony.dm [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+		"trixie_a_full",
+		"twilight",
+		"vinyl",
+		"whooves",
+		"pony",
+	)
+
+/mob/living/simple_animal/hostile/pony/Initialize(mapload)
+	icon_state = "[pick(pony_skins)]"
+	. = ..()
```
Comment (ATH1909) [inline fulp_modules/features/mobs/pony.dm:47]:
```suggestion
/mob/living/simple_animal/hostile/pony/Initialize(mapload, appearance)
	if(appearance)
		icon_state = appearance
	else
		icon_state = "[pick(pony_skins)]"
	. = ..()
```
so admins can choose which pony skin to use

there's probably a better way to do this, but I'm currently banned from #coding-general, so I can't ask to find it

**Comment by Sheits** [comment]
wtf

**Comment by SgtHunk** [comment]
Hasbro sent me a cease and desist

**Pull request closed without merging**